### DISCUSSION TITLE
Can I set the "NewCops" setting only for specific namespaces?

### DISCUSSION THREAD

**Discussion by klyonrad** [rubocop/rubocop#9680] [Q&A] [answered]
I have a project with rubocop and `rubocop-rails`.

rubocop-rails has the tendency for a very high output of new cops, lots of which are not relevant for one project.

So I tried out to set the following in the rubocop.yml

```yaml
Rails:
  NewCops: enable
```

And the pending cops warning from rubocop-rails still show up. Should this work? Was it a part of the Pending Cops feature that was considered?

**Comment by bbatsov** [comment, marked as answer]
There's currently no granularity with respect to this, but I agree it makes some sense to change it. There's also a similar conversation here https://github.com/rubocop/rubocop/pull/8565

**Reply by klyonrad** [reply]
Okay thanks a lot for the quick answer 💨 
I will not create a new ticket then for this as I think the project maintenance team will not on their own how to prioritize this 👍
